docs/develop/parent-to-dialog.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Passing data and messages to a dialog box from its host page
+description: 'Learn how to pass data to a dialog from the host page'
+ms.date: 03/11/2020
+localization_priority: Normal
+---
+
+# Passing data and messages to a dialog box from its host page
+
+Microsoft is now making available for preview new APIs for messaging from the host page (defined [here](dialog-api-in-office-add-ins.md#open-a-dialog-box-from-a-host-page)) to the dialog box.
+
+> [!Important]
+>
+> - The APIs are in preview. They are available to developers for experimentation; but they should not be used in a production add-in. Until this API is released, use the techniques described in [Pass information to the dialog box](dialog-api-in-office-add-ins.md#pass-information-to-the-dialog-box) in production add-ins.
```

Comment:
We have two "include" files for preview APIs ([using-preview-apis-host.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/includes/using-preview-apis-host.md) and [using-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/includes/using-preview-apis.md)). Could we use those here? It might also be beneficial to roll the Office 365 Insiders information into those includes.

Reply:
I've added the relevant include. As for moving the Insiders information into the includes, I'm reluctant to do that as part of this PR. There may be places where the includes are used that don't require Insiders.